### PR TITLE
I267 leroy - updating settings page

### PR DESCRIPTION
**Pull request by leoRysing**
I did some fiddling with incorporating the PageBlock and PageTitle Components into the settings page. I think the use of the components works, but I definitely think some review here would be good, as well as the usage of the created components on the page.

### CODE REVIEW

File: src/components/settings/settings.tsx [hunk 1]
```diff
@@ -7,12 +7,22 @@ import { useContext } from 'react';
 import { UserContext } from '../app/app-context/app-context';
 import './settings.module.css';
 
+import PageTitle from '../shared/page-title/page-title';
```
Comment (jamescd18) [inline src/components/settings/settings.tsx:10]: Place new imports right about the CSS file import

File: src/components/settings/settings.tsx [hunk 2]
```diff
@@ -7,12 +7,22 @@ import { useContext } from 'react';
...
+import PageBlock from '../shared/page-block/page-block';
+
+// †ødø; add the two new components, Page Title and Page Block, from shared
```
Comment (jamescd18) [inline src/components/settings/settings.tsx:13]: Update to do or remove it

File: src/components/settings/settings.tsx [hunk 3]
```diff
@@ -7,12 +7,22 @@ import { useContext } from 'react';
...
+
 const Settings: React.FC = () => {
   const user = useContext(UserContext);
+  const title = <PageTitle title="This is the Settings Page" />
```
Comment (jamescd18) [inline src/components/settings/settings.tsx:17]: Place the value of title inline with the return to simplify

File: src/components/settings/settings.tsx [hunk 4]
```diff
@@ -7,12 +7,22 @@ import { useContext } from 'react';
...
+  // const body = PageBlock("User Settings", "", "User: {user}");
```
Comment (jamescd18) [inline src/components/settings/settings.tsx:18]: Remove extraneous comments

File: src/components/settings/settings.tsx [hunk 5]
```diff
@@ -7,12 +7,22 @@ import { useContext } from 'react';
...
+  // render(<PageBlock title={'test'} headerRight={<>hi</>} body={<>hello</>} />);
+  const pageBlockBody = <>User: {user}</>;
```
Comment (jamescd18) [inline src/components/settings/settings.tsx:20]: You can inline this too

File: src/components/settings/settings.tsx [hunk 6]
```diff
@@ -7,12 +7,22 @@ import { useContext } from 'react';
...
+  const body = <PageBlock title="User Settings" headerRight={<></>} body={pageBlockBody} />
```
Comment (jamescd18) [inline src/components/settings/settings.tsx:21]: Inline this too

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Great work! It looks to be functional, there's just some code quality and cleanup items too

File: src/components/settings/settings.tsx (hunk 1 above)
Comment (leoRysing) [reply src/components/settings/settings.tsx:10]: Quick question, does this just mean make the imports a single block?

Comment (jamescd18) [reply src/components/settings/settings.tsx:9]: Yes, all the imports should be together in one block at the top of the file. The css file import is almost always the last import.

**Comment by jamescd18** [comment]
Any updates, @leoRysing?

**Review by jamescd18 — DISMISSED** [review]

**Review by jamescd18 — APPROVED** [review]
Awesome! Glad to see it!

**Pull request merged**